admin_guide/manage_nodes.adoc

```diff
@@ -10,120 +10,130 @@
 [options="nowrap"]
 ----
-$ oc describe node node2.example.com
-Name:	node2.example.com
+$ oc describe node node1.example.com
+Name:			node1.example.com
+Labels:			kubernetes.io/hostname=node1.example.com
+CreationTimestamp:	Wed, 10 Jun 2015 17:22:34 +0000
 Conditions:
-Kind    Status  LastProbeTime                   LastTransitionTime              Reason                                                              Message
-Ready   None    Tue, 03 Mar 2015 22:53:22 +0000 Tue, 03 Mar 2015 21:27:50 +0000 Node health check failed: kubelet /healthz endpoint returns not ok
+  Type		Status	LastHeartbeatTime			LastTransitionTime			Reason					Message
+  Ready 	True 	Wed, 10 Jun 2015 19:56:16 +0000 	Wed, 10 Jun 2015 17:22:34 +0000 	kubelet is posting ready status
+Addresses:	127.0.0.1
+Capacity:
+ memory:	1017552Ki
+ pods:		100
+ cpu:		2
+Version:
+ Kernel Version:		3.17.4-301.fc21.x86_64
+ OS Image:			Fedora 21 (Twenty One)
+ Container Runtime Version:	docker://1.6.0
+ Kubelet Version:		v0.17.1-804-g496be63
+ Kube-Proxy Version:		v0.17.1-804-g496be63
+ExternalID:			node1.example.com
+Pods:				(2 in total)
+  docker-registry-1-9yyw5
+  router-1-maytv
+No events.
 ----
 ====
 
-== Adding a Node
+== Adding Nodes
 You can add a node to an existing OpenShift instance using the `oc create`
-command and supplying a manifest for a node object. After the new node passes
-the health checks that are performed by the master,  pods can begin to be
+command and supplying a specification for a node object. After the newly-created
+node passes the health checks that are performed by the master, pods can be
 scheduled for the node.
 
 .To Add a Node to an Existing OpenShift Instance:
-. Create a file with the following JSON format:
-+
-.*_node.json_* file
-====
 
+. Create a specification for the new node.
+.. If you have an existing node object created, you can export the existing
+object to use as a starting point for the new node's
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-specification[specification]:
++
 ----
-{
-  "id": "node3.example.com", <1>
-  "kind": "Node", <2>
-  "apiVersion": "v1beta1", <3>
-  "labels": { <4>
-    "name": "node3",
-  },
-}
+$ oc export node <node_name> > <filename>
 ----
++
+Modify the new file to meet the desired specification for your new node.
 
-<1> Set the *`id`* parameter to the fully-qualified domain name where the node can be reached.
-This value will be shown in the `NAME` column when running the `oc get nodes`
-command.
-<2> Set the *`kind`* parameter to `Node` to identify this is a manifest for a node
-object.
-<3> Set the *`apiVersion`* parameter to the Kubernetes API version you are using.
-<4> Set any desired labels using the *`labels`* resource.
-====
+.. If you do not have an existing node object, create a new file from scratch by
+following the
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-specification[example
+specification].
```

Comment:
Are you checking the link in GH? Here's an actual build of it; seems to work for me:

http://file.rdu.redhat.com/~adellape/061115/card602_managenodes/admin_guide/manage_nodes.html#adding-nodes